src/config.ts

```diff
@@ -454,3 +454,10 @@ log.info(
   "JWT support token expiration set to %s seconds",
   JWT_SUPPORT_TOKEN_EXPIRATION
 );
+
+export const TEST_CGN_FISCAL_CODES = NonEmptyString.decode(
+  process.env.TEST_CGN_FISCAL_CODES
```

Comment:
We used a custom decoder to ensure a comma-separated list of a specific string format [here](https://github.com/pagopa/io-functions-services/blob/8f597b26ddbd07023e67469e9d849de3b563827b/utils/comma-separated-list.ts#L9)
Example:
```ts
export const TEST_CGN_FISCAL_CODES = 
  CommaSeparatedListOf(FiscalCode)
    .decode(process.env.TEST_CGN_FISCAL_CODES)
    .getOrElse([]);
```

Reply:
Please note that an invalid string format will not block the app, but it will just provide an empty array. Which may lead to unintended behaviour (you mispell something and you whitelist all users).
Maybe the following is safer:

```ts
export const TEST_CGN_FISCAL_CODES = 
  CommaSeparatedListOf(FiscalCode)
    .decode(process.env.TEST_CGN_FISCAL_CODES || "")
    .getOrElseL(err => {
      throw new Error(`Invalid TEST_CGN_FISCAL_CODES value: ${readableReport(err)}`
    });
```